cpp/net/connection_pool.cc

```diff
@@ -449,7 +449,7 @@ unique_ptr<ConnectionPool::Connection> ConnectionPool::Get(const URL& url) {
                                     kZeroMillis};
     evhtp_connection_set_timeouts(handle->connection(), &read_timeout,
                                   &write_timeout);
-    evhtp_set_hook(&handle->connection()->hooks, evhtp_hook_on_conn_error,
+    evhtp_set_hook(&handle->connection()->hooks, evhtp_hook_on_error,
```

Comment:
No; they added `evhtp_hook_on_conn_error` in `1.2.10`, the FreeBSD `ports/www/libevhtp` (at least on my `10.2-RELEASE` machine) is at `1.2.9`.